src/hats.ts

```diff
@@ -108,13 +118,15 @@ export function handleHatEligibilityChanged(
   hat.eligibility = event.params.newEligibility.toHexString();
   hat.save();
 
-  // create new HatEligibilityChangedEvent 
-  let hatEligibilityChangedEvent = new HatEligibilityChangedEvent(createEventID(event, 'HatEligibilityChanged'));
+  // create new HatEligibilityChangedEvent
+  let hatEligibilityChangedEvent = new HatEligibilityChangedEvent(
+    createEventID(event, "HatEligibilityChanged")
+  );
   hatEligibilityChangedEvent.blockNumber = event.block.number.toI32();
   hatEligibilityChangedEvent.timestamp = event.block.timestamp;
   hatEligibilityChangedEvent.transactionID = event.transaction.hash;
-  hatEligibilityChangedEvent.hatId = hatIdToHex(event.params.hatId);
```

Comment:
you're removing this because its duplicative with `.hat`, correct?

Reply:
yes